### PR TITLE
complete vignette and remove pkg dependencies

### PR DESCRIPTION
**Pull request by mianzg**

### CODE REVIEW

**Review by larnsce — COMMENTED** [review]
Great job! This is a well written vignette now.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,14 +1,16 @@
 Package: washr
-Title: Efficient workflow tools for openwashdata package
+Title: R Data Package Development Tools For Openwashdata
```
Comment (larnsce) [inline DESCRIPTION:2]: @mianzg: there is no need to capitalize each work. You can keep it lower case;

R data package development tools for openwashdata

**Comment by larnsce** [comment]
Well done 🚀 

Merge once you have revised my one comment.

File: DESCRIPTION (hunk 1 above)
Comment (mianzg) [reply DESCRIPTION]: It has to be Title Case, required by CRAN

Comment (larnsce) [reply DESCRIPTION]: washr: R Data Package Development Tools for Water Sanitation and Hygiene Data

**Pull request closed without merging**